CHANGELOG.md

```diff
@@ -13,7 +13,9 @@ This release extends support to OneView Rest API version 1800 (OneView v5.30) fo
 - Hypervisor Cluster Profile
 - Hypervisor Manager
 - Interconnects
+- Interconnect Types
```

Comment:
why did u add this resource when there is no lib files related to interconnect types.?

Reply:
its covered in Interconnects resource itself. We had it separately in previous release as well. Hence followed the same